### ISSUE TITLE
Enable multiple scripts for cli.js

### ISSUE THREAD

**Issue by TimHenkelmann** [MusicConnectionMachine/StructuredData#76]
At the moment, the cli only takes one script as input parameter.
Adjust it, such that 
```
node cli.js -s dbpedia worldcat musicbrainz
```
or 
```
node cli.js -s all
```
is possible.

**Comment by TimHenkelmann** [comment]
Implemented in #78 
The input parameters changed such that one can now execute the scripts like
```
node cli.js dbpedia worldcat
```
(scrapes dbpedia and worldcat)
or
```
node cli.js all
```
(scrapes dbpedia, musicbrainz [and worldcat+allmusic as soon as they are added])

**Comment by kordianbruck** [comment]
Good thinking @TimHenkelmann - excellent implementation. Makes much more sense this way.
